dataframes.py

```diff
@@ -88,13 +91,12 @@ def combine_with_prices(df_amm, df_weth_prices, df_cow_prices):
             "profit_vs_holding_change"
         )
     )
-
     return df_amm_aggregate
 
 
 def plot_profit_vs_holding(df):
     plt = (
-        df.filter(~(pl.col("total_value_change").log().abs() > 2))
+        df.filter(~(pl.col("total_value_change").log().abs() > 0.02))
```

Comment:
This should filter out all liquidity injections:
```suggestion
        df.filter(
            ((pl.col(TOKEN0).diff() == 0) & (pl.col(TOKEN1).diff() == 0))
            | (
                (pl.col(TOKEN0).diff() != 0)
                & (pl.col(TOKEN1).diff() != 0)
                & (pl.col(TOKEN0).diff() * pl.col(TOKEN1).diff() < 0)
            )
        )
```